src/rancilio-pid.cpp

```diff
@@ -2022,6 +2039,7 @@ void looppid() {
     if ((millis() - lastTempEvent) > tempEventInterval) {
         //send temperatures to website endpoint
         sendTempEvent(temperature, brewSetPoint, pidOutput);
+        sendWeightEvent(weight, weightBrew, 22.22);
```

Comment:
why is it sending the fixed value 22.22?

Reply:
that is my fault, that was just for testing. Should be weightPreBrew